src/scss/components/_thumbnail.scss

```diff
@@ -10,22 +10,22 @@
 **/
 
 .thumbnail {
```

Comment:
@eduardojmatos In cases where we use a mixin or placeholder, will they come before?